projects/packages/jetpack-mu-wpcom/package.json

```diff
@@ -48,9 +49,11 @@
 	"dependencies": {
 		"@automattic/calypso-color-schemes": "3.1.3",
 		"@automattic/color-studio": "4.0.0",
+		"@automattic/components": "2.2.0",
 		"@automattic/i18n-utils": "1.2.3",
 		"@automattic/jetpack-base-styles": "workspace:*",
 		"@automattic/jetpack-shared-extension-utils": "workspace:*",
+		"@automattic/launchpad": "1.1.0",
```

Comment:
Here too, whatever is using `i18n-calypso` in here is probably not going to be translated properly.